app/codec.go

```diff
@@ -0,0 +1,17 @@
+package app
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/std"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func MakeCodecs() (*std.Codec, *codec.Codec) {
```

Comment:
This is getting rather complex. Every iteration of the sdk seems to make it harder to use